mocks/routes/zorgned.js

```diff
@@ -1,25 +1,32 @@
-const ZORGNED_JZD_RESPONSE = require('../fixtures/zorgned-jzd.json');
-const ZORGNED_AV_RESPONSE = require('../fixtures/zorgned-av.json');
+const ZORGNED_AV_AANVRAGEN_RESPONSE = require('../fixtures/zorgned-av-aanvragen.json');
+const ZORGNED_JZD_AANVRAGEN_RESPONSE = require('../fixtures/zorgned-jzd-aanvragen.json');
+const ZORGNED_AV_PERSOONSGEGEVENSNAW_RESPONSE = require('../fixtures/zorgned-av-persoonsgegevensNAW.json');
 const settings = require('../settings.js');
 
 module.exports = [
   {
-    id: 'get-zorgned-jzd',
+    id: 'post-zorgned-aanvragen',
     url: `${settings.MOCK_BASE_PATH}/zorgned/aanvragen`,
     method: 'POST',
     variants: [
       {
         id: 'standard',
-        type: 'json',
+        type: 'middleware',
         options: {
-          status: 200,
-          body: ZORGNED_JZD_RESPONSE,
+          middleware: (req, res, next, core) => {
+            res.status(200);
+            return res.send(
+              req.headers['x-mams-api-user'] === 'AV'
```

Comment:
core.logger.errors('error bericht') voor als de header deze key niet bevat? Als je denkt dat dat nodig is.